common_sleep.Rmd

```diff
@@ -21,30 +22,34 @@ library(lme4)
 library(arm)
 library(ggplot2)
 library(reshape2)
+library(magrittr)
+library(tidyr)
+library(pander)
 
 ggplot2::theme_set(theme_classic())
 ```
 
-## Presentation of the case study
-We are analysing the effect of sleep deprivation on reaction time of long distance truck drivers. There are 18 subjects in the dataset and for each subject, the reaction time was measured for 10 days. The subjects were allowed only a limited amount of sleep for these 10 subsequent days.Each subject's reaction time was measured several times on each day of the trial and an average was taken. 
+Laura Bogeart, Ben De Maesschalck, Bianca Florenzi, Jonas Jonker, Nina Rank, Alexandra Stanciu, Maria Tsontaki, Dries Vrijens
```

Comment:
Unless there is a good reason to. Authors should be part of the meat data. (It also prints nicer)

Reply:
Can this be changed to put all names in a nice way?

Reply:
I think so, i will look into that